FitNesseRoot/FitNesse/UserGuide/content.txt

```diff
@@ -13,6 +13,7 @@
  * [[Working with the FitNesse wiki][>FitNesseWiki]]
  * [[Writing Acceptance Tests][>WritingAcceptanceTests]] (jump directly to [[Fit][>WritingAcceptanceTests.FitFramework]] or [[Slim][>WritingAcceptanceTests.SliM]])
  * [[Administering FitNesse][>AdministeringFitNesse]]
+ * [[Extend Fitnesse with Plugins][.PlugIns]]
```

Comment:
I don't think the name `Extend FitNesse with Plugins` covers it.

Maybe `Installed plugins`?